commons/commands.py

```diff
@@ -497,6 +497,7 @@
 K8S_CONSUL_UPDATE_CMD = 'kubectl exec -it {} -c {} -- {}'
 GET_STATS = "consul kv get -recurse stats"
 GET_BYTECOUNT = "consul kv get -recurse bytecount"
+GET_NAMESPACE = "kubectl config set-context --current --namespace=default"
```

Comment:
This is not get command, its set command. And, make it paramerterized like below:
kubectl config set-context --current --namespace={}